modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,156 @@
+	machine.openshift.io/cluster-api-machine-type: master
+  name: kni1-master-3
+  namespace: openshift-machine-api
+spec:
+  providerSpec:
+	value:
+  	hostSelector: {}
+  	image:
+    	checksum: http://172.22.0.3:6180/images/rhcos-44.81.202004250133-0-openstack.x86_64.qcow2/rhcos-44.81.202004250133-0-compressed.x86_64.qcow2.md5sum
+    	url: http://172.22.0.3:6180/images/rhcos-44.81.202004250133-0-openstack.x86_64.qcow2/rhcos-44.81.202004250133-0-compressed.x86_64.qcow2
+  	userData:
+    	name: master-user-data
+EOF
+----
+
+[NOTE]
+====
+The baremetalhost, secret and machine objects can be created at once in a single command or yaml file if needed.
+====
+
+. The provisioning process (via the baremetal-operator) will install RHCOS and prepare the host to be added to the cluster:
+
+[source,terminal]
+----
+$ oc get bmh -A
+----
+
+.Example output
+[source,terminal]
+----
+NAMESPACE           	  NAME        	 STATUS PROVISIONING STATUS  	  CONSUMER        BMC                  	  HARDWARE PROFILE   ONLINE   ERROR
+openshift-machine-api   kni1-master-0  OK   	provisioned          	  kni1-worker-0   ipmi://10.19.136.22     dell-raid          true	 
+openshift-machine-api   master-1   	   OK   	externally provisioned  kni1-master-1   ipmi://172.22.0.1:6231                     true	 
+openshift-machine-api   master-2   	   OK   	externally provisioned  kni1-master-2   ipmi://172.22.0.1:6232                     true	 
+openshift-machine-api   master-3   	   OK   	provisioning            kni1-master-3   ipmi://172.22.0.1:6233  unknown            true	 
+openshift-machine-api   worker-0    	 OK   	provisioned          	  kni1-worker-1   ipmi://10.19.143.29  	  dell-raid          true
+----
+
+. The node will be added to the cluster:
+
+[source,terminal]
+----
+$ oc get bmh -A
+----
+
+.Example output
+[source,terminal]
+----
+NAMESPACE           	  NAME    	STATUS  PROVISIONING STATUS  	  CONSUMER          BMC                  	  HARDWARE PROFILE   ONLINE   ERROR
+openshift-machine-api   master-1  OK   	  externally provisioned  kni1-master-1     ipmi://172.22.0.1:6231                     true	 
+openshift-machine-api   master-2  OK    	externally provisioned  kni1-master-2     ipmi://172.22.0.1:6232                     true	 
+openshift-machine-api   master-3  OK    	provisioned          	  kni1-master-3     ipmi://172.22.0.1:6233   unknown           true	 
+openshift-machine-api   worker-0	OK    	provisioned          	  kni1-worker-0     ipmi://10.19.143.29  	   dell-raid         true
+----
+
+. Confirm that the node has been added to the cluster:
+
+[source,terminal]
+----
+$ oc get nodes
+----
+
+.Example output
+----
+NAME                  STATUS      ROLES      AGE	  VERSION
+kni1-master-1        	available	  master     141m   v1.18.2
+kni1-master-2         available	  master     141m   v1.18.2
+kni1-master-3         available	  master     4m2s   v1.18.2
+kni1-worker-0         available	  worker     87m	  v1.18.2
+----
```

Comment:
Adjust the output to render correctly in the preview. Presently it is a bit distorted.